tests/merkleAPI.ts

```diff
@@ -639,7 +645,8 @@ if (privateKey !== undefined && privateKey !== "") {
       });
     });
 
-    describe("signer requests", function () {
+    // See issue #653
+    describe.skip("signer requests", function () {
```

Comment:
I noticed issues with signer requests but only for submits. Are the requests to fetch existing ones failing too?

What I saw in the logs
```
submitMessage error: signer add/remove messages are not supported on migrated hubs
```

Reply:
Also, this is relatively new. I was able to create a signer on our hub recently, but not anymore.

Reply:
I'm not sure. The test for fetching an existing one attempts to use the token it just created. I suppose we could possibly create a long-lived token outside of this library. But then it would probably be better to fix the code that creates the tokens in this library instead